vm/isolated/isolated.go

```diff
@@ -172,29 +190,39 @@ func (inst *instance) ssh(command string) error {
 }
 
 func (inst *instance) repair() error {
-	log.Logf(2, "isolated: trying to ssh")
-	if err := inst.waitForSSH(30 * time.Minute); err == nil {
-		if inst.cfg.TargetReboot {
-			log.Logf(2, "isolated: trying to reboot")
-			inst.ssh("reboot") // reboot will return an error, ignore it
-			if err := inst.waitForReboot(5 * 60); err != nil {
-				log.Logf(2, "isolated: machine did not reboot")
+	if inst.cfg.TargetReboot {
```

Comment:
Currently we do `inst.waitForSSH(30 * time.Minute)` even if TargetReboot is not set. With this change we seem to skip it, please preserve the current behavior.